### PR TITLE
add some games

### PR DESCRIPTION
**Pull request by Dei657**

### CODE REVIEW

File: gamedb.yaml [hunk 1]
```diff
@@ -66,6 +78,20 @@
   patches:
     - exec: rm -f installscript.vdf
 
+- name: Borderlands 2
+  platform: epic-store
+  id: Dodo
+  banner: steam:49520
+  launch_options: -nolauncher
```
Comment (alkazar) [inline gamedb.yaml:85]: does this actually work? I wouldn't expect it to.

Comment (Dei657) [reply gamedb.yaml:85]: I checked several times, two games were launched without a launcher, I'm sure that this command is working.

File: gamedb.yaml [hunk 2]
```diff
@@ -182,6 +214,18 @@
   banner: steam:612390
   status: verified
 
+- name: Deponia The Complete Journey
+  platform: epic-store
+  id: 03b7b25a77024cc0adf44777f791ffc4
+  banner: steam:214340
```
Comment (alkazar) [inline gamedb.yaml:220]: I think this should be steam:292910
214340 is just plain Deponia

File: gamedb.yaml [hunk 3]
```diff
@@ -261,12 +326,30 @@
...
 - name: GoNNER
   platform: epic-store
   id: Emperor
   banner: steam:437570
   status: verified
 
+- name: Guild of Dungeoneering
+  platform: epic-store
+  id: b8bdc716e79e461bbf161d99f842f29a
+  banner: steam:317820
```
Comment (alkazar) [inline gamedb.yaml:350]: This is the ultimate edition on Steam, is the version on EGS here also the ultimate edition?

Comment (Dei657) [reply gamedb.yaml:350]: it's a standard edition game

**Comment by alkazar** [comment]
Awesome, thanks!

**Pull request merged**